ansible/roles/backups/templates/backup_snapshots.py.j2

```diff
@@ -13,6 +13,11 @@ else:
     raise Exception('Usage is backup_snapshots.py filename <s3_bucket> [ <endpoint> ]')
 
 filename = sys.argv[1]
+# This will give daily / weekly
+upload_filename = sys.argv[1].split('_')[1]
```

Comment:
this relies on the format of the input filename which isn't obvious from outside this script. It might be better to pass in the upload filename as an additional param.